docs/usage/storefront.md

```diff
@@ -0,0 +1,38 @@
+# Make a Storefront API call
+
+The library also allows you to send GraphQL requests to the [Shopify Storefront API](https://shopify.dev/docs/storefront-api). To do that, you can use the `StorefrontClient` using the current shop URL and an `accessToken`.
+
+You can obtain Storefront API access tokens for both private apps and sales channels. Please read [our documentation](https://shopify.dev/docs/storefront-api/getting-started) to learn more about Storefront Access Tokens.
```

Comment:
Do we want to also provide an example like you have in the PR comment showing how to request this information from the REST API? Or do we feel like directing them to the docs is enough? I think giving the example post request for the token might be useful to keep everything in the same place and show explicitly how to make that request using the library clients.

Reply:
Great point, I think that's helpful. I've added the example for the PR there.